src/pages/Background/index.js

```diff
@@ -197,47 +343,73 @@ class PpdnsBackground {
       // nothing necessary here, but required before Chrome 42
     });
 
-    try {
-      if (!chrome.notifications.onClicked.hasListeners()){
-        chrome.notifications.onClicked.addListener(async (notificationId) => {
-          console.debug('Notification clicked: %s', notificationId);
-          await chrome.tabs.create({ url: 'https://polyswarm.network/account/api-keys' }).then(
-            tab => { console.info('Tab opened in Polyswarm website: %s', tab); }
-          );
-        });
-      }
-    }catch {
-      console.warn('Could not set onClick handlers');
+    await this.snoozeNotifications(snoozedUntil);
+  }
+
+  async getLastSuccessString() {
+    let apiKeyCheckedDate = '0'+(await this.storage.get(SETTINGS_KEY))[SETTINGS_KEY].apiKeyCheckedDate;
+    if (Number(apiKeyCheckedDate) >= Date.now() - 86400000){  // yesterday
+      return 'today'
+    }else{
+      let lastDate = new Date(Number(apiKeyCheckedDate));
+      return 'by ' + lastDate.toLocaleDateString()
```

Comment:
Native english speakers, please double check this piece.

The idea is to return `today` or `by 2024/12/20` or what makes sense in the browser language settings.

That will turn into `last time it worked was by 2024...` or `last time it worked was today.`
If this is not correct in English, please provide a suggestion for the `by ` thing.

Reply:
@supernothing & @mrtizmoatwork I am marking you here to be able to check the English spellings.

Reply:
```suggestion
      return 'on ' + lastDate.toLocaleDateString()
```

I think this sounds right:
>last time it worked was on 2024/12/20